SQL/0000-00-00-schema.sql

```diff
@@ -762,6 +802,33 @@ CREATE TABLE `mri_protocol_checks` (
     FOREIGN KEY (`Scan_type`) REFERENCES `mri_scan_type` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `mri_protocol_checks_group` (
+    `MriProtocolChecksGroupID`   INT(2) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `Name`                       VARCHAR(255)    NOT NULL,
+    PRIMARY KEY (`MriProtocolChecksGroupID`)
+) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+
+INSERT INTO `mri_protocol_checks_group` VALUES(1, 'Default MRI protocol checks group');
+
+CREATE TABLE `mri_protocol_checks_group_rel` (
```

Comment:
Is there a value in having this table allow for duplicate values? Should the (MriProtocolChecksGroupID, MriProtocolChecksID) pair be a primary key?

Reply:
In fact, I want to make sure that each line of table `mri_protocol` is associated to at most one group, so yeah I guess your suggestion makes sense. Should be corrected in the next commit.